client/components/Item/SeasonTabs.vue

```diff
@@ -88,3 +91,9 @@ export default Vue.extend({
   }
 });
 </script>
+
+<style lang="scss">
```

Comment:
I guess this should be scoped? :)
```suggestion
<style scoped lang="scss">
```